drivers/hmis_csv_importer/spec/models/importer/twenty_twenty/force_valid_coc_spec.rb

```diff
@@ -55,7 +55,7 @@ def setup(with_cleanup:)
     end
 
     import_hmis_csv_fixture(
-      'drivers/hmis_csv_importer/spec/fixtures/files/twenty_twenty/cleanup_move_ins',
+      'drivers/hmis_csv_importer/spec/fixtures/files/twenty_twentytwo/cleanup_move_ins',
```

Comment:
Should we have separate fixtures for each extension?

Reply:
Hmm, I bungled this a bit.  Pushed a fix with new fixture and tests to cover the case where we can't determine the CoC.